changing bootstrap col-xs to col-md

MiczFlor opened the pull request:
Better for phones

aerkalov commented:
No need to send each file in separate Pull Request. I cherry picked the commits, rebased, rewrote commit message and merged it.

MiczFlor commented:
My bad. I need to learn github

Am Dienstag, 3. November 2015 schrieb Aleksandar Erkalović :

> No need to send each file in separate Pull Request. I cherry picked the
> commits, rebased, rewrote commit message and merged it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcefabric/Booktype/pull/452#issuecomment-153207281>
> .
>


-- 

Micz Flor
Co-Founder and Director of Business
micz.flor@sourcefabric.org
www.sourcefabric.org

Prinzessinnenstrasse 20
10969 Berlin, Germany
Phone: +49 (0) 30 616 29 281
Fax: +49 (0) 30 616 29 283

Registergericht: Amtsgericht Charlottenburg / HRB 149568B
USt: DE288649893
Managing Directors: Sava Tatić, Fabienne Riener
